Musician Module/MusicianModule.cs

```diff
@@ -0,0 +1,512 @@
+
+        /// <summary>
+        /// Define the settings you would like to use in your module.  Settings are persistent
+        /// between updates to both Blish HUD and your module.
+        /// </summary>
+        protected override void DefineSettings(SettingsManager settingsManager)
+        {
+        }
+
+        /// <summary>
+        /// Allows your module to perform any initialization it needs before starting to run.
+        /// Please note that Initialize is NOT asynchronous and will block Blish HUD's update
+        /// and render loop, so be sure to not do anything here that takes too long.
+        /// </summary>
+        protected override void Initialize()
+        {
+            Directory.CreateDirectory(Path.Combine(GameService.FileSrv.BasePath, "musician"));
+            xmlParser = new XmlMusicSheetReader();
+            displayedSheets = new List<SheetButton>();
+        }
+
+        /// <summary>
+        /// Load content and more here. This call is asynchronous, so it is a good time to
+        /// run any long running steps for your module. Be careful when instancing
+        /// <see cref="Blish_HUD.Entities.Entity"/> and <see cref="Blish_HUD.Controls.Control"/>.
+        /// Setting their parent is not thread-safe and can cause the application to crash.
+        /// You will want to queue them to add later while on the main thread or in a delegate queued
+        /// with <see cref="Blish_HUD.DirectorService.QueueMainThreadUpdate(Action{GameTime})"/>.
+        /// </summary>
+        protected override async Task LoadAsync()
+        {
+            MusicianTab = GameService.Director.BlishHudWindow.AddTab("Musician", ICON, BuildHomePanel(GameService.Director.BlishHudWindow), 0);
+        }
+
+        /// <summary>
+        /// Allows you to perform an action once your module has finished loading (once
+        /// <see cref="LoadAsync"/> has completed).  You must call "base.OnModuleLoaded(e)" at the
+        /// end for the <see cref="ExternalModule.ModuleLoaded"/> event to fire and for
+        /// <see cref="ExternalModule.Loaded" /> to update correctly.
+        /// </summary>
+        protected override void OnModuleLoaded(EventArgs e)
+        {
+
+            base.OnModuleLoaded(e);
+        }
+
+        /// <summary>
+        /// Allows your module to run logic such as updating UI elements,
+        /// checking for conditions, playing audio, calculating changes, etc.
+        /// This method will block the primary Blish HUD loop, so any long
+        /// running tasks should be executed on a separate thread to prevent
+        /// slowing down the overlay.
+        /// </summary>
+        protected override void Update(GameTime gameTime)
+        {
+            if (settingBackgroundPlayback.Value && GameService.GameIntegration.Gw2IsRunning)
+            {
+                if (Utils.Window.GetForegroundWindow() != GameService.GameIntegration.Gw2Process.MainWindowHandle)
```

Comment:
This namespace was updated and it'd probably be better if you did `WindowUtil.OnTop` which will let you know if Blish HUD is currently on top or not.  `GetForegroundWindow` is no longer exposed outside of the static util class.

Looking at it now, it may be best for that `OnTop` property to get moved to the `GameIntegrationService` at some point.